caffe2/__init__.py

```diff
@@ -0,0 +1,6 @@
+import warnings
+from torch.onnx import _CAFFE2_ATEN_FALLBACK
+
+if not _CAFFE2_ATEN_FALLBACK:
```

Comment:
AFAIK this is the only flag available in python realm to signal caffe2 is built

From https://github.com/pytorch/pytorch/blob/master/torch/csrc/onnx/init.cpp#L198-L202

```
#ifdef BUILD_CAFFE2
  onnx.attr("_CAFFE2_ATEN_FALLBACK") = true;
#else
  onnx.attr("_CAFFE2_ATEN_FALLBACK") = false;
#endif
```

Reply:
Hm, why not use `from caffe2.proto import caffe2_pb2` as an indicator (because this assumes that caffe2 always must be build with torch, which is not the case)